helpers.py

```diff
@@ -0,0 +1,261 @@
+"""
```

Comment:
lets figure out what functions are duplicates with what are within akita_inu_asa_utils.py, rob them and delete the duplicates to keep things better organized